filebeat/input/file_other.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 type FileStateOS struct {
-	Inode  uint64 `json:"inode,omitempty"`
-	Device uint64 `json:"device,omitempty"`
+	Inode  uint64 `json:"inode,"`
```

Comment:
mostly for sanity checks. If registry file is copied and loaded you won't see any errors, but Inode/Device fields initialized to 0.